native/screens/SignUp.jsx

```diff
@@ -1,180 +1,309 @@
+            emailInputRef.current &&
+            emailInputRef.current.focus()
+          }}
+        />
+      </View>
+      <View style={styles.inputContainer}>
+        <Text>Email Address</Text>
+        <TextInput
+          value={newUser.email}
+          style={styles.input}
+          onChangeText={(email) => setNewUser((prevState) => {
+            return {
+              ...prevState,
+              email: email
+            }
+          })}
+          keyboardType="email-address"
+          autoCapitalize="none"
+          ref={emailInputRef}
+          onFocus={()=>setEmailError(null)}
+          onBlur={() => handleEmailCheck()}
+          returnKeyType="next"
+          onSubmitEditing={() => {
+            passwordInputRef.current &&
+            passwordInputRef.current.focus()
+          }}
+        />
+        <Text>{emailError}</Text>
+      </View>
+      <View style={styles.inputContainer}>
+        <Text>Password</Text>
+        <TextInput 
+          value={newUser.password}
+          style={styles.input}
+          onChangeText={(password) => setNewUser((prevState) => {
+            return {
+              ...prevState,
+              password: password
+            }
+          })}
+          ref={passwordInputRef}
+          returnKeyType="next"
+          secureTextEntry={true}
+          onSubmitEditing={() => {
+            confirmPasswordInputRef.current &&
+            confirmPasswordInputRef.current.focus()
+          }}
+        />
+      </View>
+      <View style={styles.inputContainer}>
+        <Text>Re-enter Password</Text>
+        <TextInput 
+          value={newUser.confirm_password}
+          style={styles.input}
+          onChangeText={(confirmPassword) => setNewUser((prevState) => {
+            return {
+              ...prevState,
+              confirm_password: confirmPassword
+            }
+          })}
+          ref={confirmPasswordInputRef}
+          returnKeyType="next"
+          secureTextEntry={true}
+          onSubmitEditing={() => {
+            confirmPasswordInputRef.current &&
+            confirmPasswordInputRef.current.focus()
+          }}
+        />
+        { newUser.password !== "" && 
+        <Text>
+          {newUser.password === newUser.confirm_password ? 
+            "Passwords match." : "Passwords do not match." }
+        </Text> }
+      </View>
+      {/* Replace with SingleButton Component */}
+      <TouchableOpacity
+        style={styles.singleButton}
+        onPress={() => handleSignUp()}
+        color="white">
+        <Text style={styles.buttonText}>Register</Text>
+      </TouchableOpacity>
+      <View>
+        <Text style={styles.inlineText}>
+          Already have an account? 
+        <Text style={styles.link} onPress={() => navigation.navigate('SignIn')}>
+          Sign in.</Text>
+        </Text>
+      </View>
+    </View>
+  );
+};
+
+const styles = StyleSheet.create({
+  modalContainer: {
+    height: '100%',
```

Comment:
please try to alphabetize your properties & classes; it will pay off in the long run in case we ever have to come back to update the styling